src/librustc_mir/transform/cleanup_post_borrowck.rs

```diff
@@ -54,8 +54,10 @@ impl MirPass for CleanupPostBorrowck {
             delete.visit_mir(mir);
         }
 
-        let mut delete = DeleteUserAssertTy;
-        delete.visit_mir(mir);
+        if !tcx.sess.opts.debugging_opts.include_user_assert_ty {
```

Comment:
Also, it seems like this flag isn't quite what it suggests -- that is, it doesn't "include it" in the dump, it "stops deleting it". Can't you just use `-Ζdump-mir` and inspect the results from earlier passes? For example, I generally do `-Zdump-mir=nll`

Reply:
I completely forgot I could view the MIR from earlier passes, so yeah, I'll remove this flag.

Reply:
Removed this.